tests/integration/jsonschema/jsonschema.go

```diff
@@ -0,0 +1,70 @@
+package jsonschema
```

Comment:
File copied from the `integration-testing` private repo. Without any modification from my side.